lib/bashcov/detective.rb

```diff
@@ -38,8 +38,6 @@ def shellscript?(filename)
         (shellscript_extension?(filename) && shellscript_syntax?(filename))
     end
 
-  private
```

Comment:
Nit: `#shellscript_syntax?` could have been left as private since we don't call/test it directly but that doesn't really matter.

Reply:
Whoops; happy to fix this.